TypeCobol.Test/Parser/CodeElements/HeaderCodeElements.rdz.cbl

```diff
@@ -6,7 +6,7 @@
 000060 FILE-CONTROL.
 000070 FILE SECTION.
 000080 INPUT-OUTPUT SECTION.
-000090 I-O-CONTROL.
+000090      I-O-CONTROL.
```

Comment:
I-O-CONTROL must starts in Area A. I forgot that in the issue.
Can you update IOControlParagraphHeader ?